metadata/multirepo/multirepo.go

```diff
@@ -311,6 +311,11 @@ func (client *MultiRepoClient) DownloadTarget(repos []string, targetFile *metada
 	log := metadata.GetLogger()
 
 	for _, repoName := range repos {
+		if _, ok := client.TUFClients[repoName]; !ok {
```

Comment:
Would you mind storing the client as a variable, and then use it below on line 320 and 330 to avoid referencing the map again, would be slightly more readable.